app/helpers/uploads_helper.rb

```diff
@@ -9,7 +9,8 @@ class Postprocessor
     public
     def self.create_handler(name)
       name = name.to_s
-      PROCS[name] = lambda 
+      # from http://blog.gsamokovarov.com/ruby-tips-tricks-1, this works despite the warning
```

Comment:
Read the linked article: apparently, Ruby simply throws up a warning message for exactly this use case, despite it being intended behavior.  I don't get it, but oh well.